test/unit/metric/normalizer-tx-segment.test.js

```diff
@@ -5,28 +5,27 @@
 
 'use strict'
 
-const tap = require('tap')
-
+const test = require('node:test')
+const assert = require('node:assert')
 const TxSegmentNormalizer = require('../../../lib/metrics/normalizer/tx_segment')
 const txTestData = require('../../lib/cross_agent_tests/transaction_segment_terms')
 
-tap.test('The TxSegmentNormalizer', (t) => {
+test('The TxSegmentNormalizer', async (t) => {
   // iterate over the cross_agent_tests
   txTestData.forEach((test) => {
     // create the test and bind the test data to it.
-    t.test(`should be ${test.testname}`, (t) => {
-      runTest(t, test)
+    t.test(`should be ${test.testname}`, () => {
+      runTest(test)
     })
   })
```

Comment:
For me, it would be easier to understand that these tests are doing what is intended if the loop is refactored to:

```js
for (const test of txTestData) {
	await t.test(`should be ${test.testname}`, () => {
		runTest(test)
	})
}
```